sdk/database/dbplugin/v5/grpc_client_test.go

```diff
@@ -134,15 +134,6 @@ func TestGRPCClient_NewUser(t *testing.T) {
 			doneCtx:   runningCtx,
 			assertErr: assertErrNotNil,
 		},
-		"bad expiration": {
-			client: fakeClient{},
-			req: NewUserRequest{
-				Password:   "njkvcb8y934u90grsnkjl",
-				Expiration: invalidExpiration,
-			},
-			doneCtx:   runningCtx,
-			assertErr: assertErrNotNil,
-		},
```

Comment:
In case anyone is wondering why this test case is removed: protobuf now supports a much larger range for durations than Go's time.Duration:

https://github.com/golang/protobuf/blob/75de7c059e36b64f01d0dd234ff2fff404ec3374/ptypes/timestamp.go#L16-L23

vs.

> A Duration represents the elapsed time between two instants as an int64 nanosecond count. The representation limits the largest representable duration to approximately 290 years.
- https://pkg.go.dev/time#Duration